clusterDeployer.py

```diff
@@ -107,6 +178,8 @@ def teardown(self) -> None:
         logger.info(f"Tearing down {cluster_name}")
         self._ai.ensure_cluster_deleted(self._cc.name)
 
+        self.update_etc_resolvconf(restore=True)
```

Comment:
This currently breaks fresh deployments (no clusters running yet).  It tries to get the API VIP:
```
api_vip = self._ai.get_ai_cluster_info(cluster_name).api_vip
```
But internally this sys.exits because the AI cluster named `cluster_name` does not exist.

Reply:
ah right. Should be fixed now. Thanks.